web/modules/custom/modal_form_example/src/Form/Demo.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace Drupal\modal_form_example\Form;
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\node\Entity\Node;
+use Drupal\file\Entity\File;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * A class to create form for creating node.
+ */
+class Demo extends FormBase {
+
+  /**
+   * To Get the Current user details.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $account;
+
+  /**
+   * To Load the file to database.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Class constructor.
+   */
+  public function __construct(AccountInterface $account, EntityTypeManagerInterface $entity_type_manager) {
+    $this->account = $account;
+    $this->entityTypeManager = $entity_type_manager;
+
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    // Instantiates this form class.
+    return new static(
+      // Load the service required to construct this class.
+      $container->get('current_user'),
+      $container->get('entity_type.manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $step = 1;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'multi_step_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $uid = $this->account->id();
+
+    if ($uid) {
+      $user = $this->entityTypeManager->getStorage('user')->load($uid);
+      $pictureUri = $user->user_picture->entity->getFileUri();
+      $name = $user->getAccountName();
+      $picture = str_replace(
+            'public://pictures/2022-01/',
+            '',
+            $pictureUri
+        );
+    }
+
+    // Add a wrapper div used by the Form API to update the form using AJAX.
+    $form['#prefix'] = '<div id="ajax_form_multistep_form">';
+    $form['#suffix'] = '</div>';
+    if ($this->step == 1) {
+      $form['message-step'] = [
+        '#markup' =>
+        '<div class="step">' . $this->t('Step 1 of 2') . '</div>',
+      ];
+      $form['my_file'] = [
+        '#type' => 'managed_file',
+        '#title' => '<div class ="contain1">
```

Comment:
Convert this to a render array, do not use html here directly in the `#title` key.